src/main/java/com/jcabi/ssh/SSHByPassword.java

```diff
@@ -124,7 +82,7 @@ public int exec(final String command, final InputStream stdin,
         randomize = true,
         types = IOException.class
     )
-    private Session session() throws IOException {
+    public Session session() throws IOException {
```

Comment:
@dmzaytsev it was transfered to AbstractSSHShell. So they need to me protected or public.